app/models/playlist.js

```diff
@@ -1,12 +1,12 @@
 import Backbone from 'backbone';
-
+import cozysdk from 'cozysdk-client';
 
 const Playlist = Backbone.Model.extend({
     
     defaults: {
         _id: undefined,
         title: '',
-        tracks: '',
+        tracks: [],
```

Comment:
Be careful with array or object assigments in defaults: they are shared objects in JS, so each instances of the _Playlist_ model will all share the same `tracks` array, which is probably not what you want.

To workaround this, assign the `this.tracks